ignite/contrib/metrics/regression/canberra_metric.py

```diff
@@ -24,7 +24,7 @@ def reset(self):
 
     def _update(self, output):
         y_pred, y = output
-        errors = torch.abs(y.view_as(y_pred) - y_pred) / (y_pred + y.view_as(y_pred))
+        errors = torch.abs(y.view_as(y_pred) - y_pred) / (torch.abs(y_pred) + torch.abs(y.view_as(y_pred)))
```

Comment:
Please, update the docstring formula and also add a link to sklearn.

Reply:
@vfdev-5 done

Reply:
How about the link to sklearn ?